CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 All notable changes to this project will be documented in this file.
 
+## [8.23.0](https://github.com/openvoxproject/openvox/tree/8.23.0) (2025-09-06)
+
+Beyond the changes noted below, this release contains the following:
```

Comment:
Did you write this by hand? the generator will overwrite it on the next release. but you can make persistent notes:
* create a 8.23.0 milestone
* create a new issue, assign it to the milestone
* add `release-summary` label to it
* add the next to the issue
* close issue and milestone

the generator will pick up the text and add it here.

Reply:
Ah nice, thank you!

Reply:
Didn't seem to work :/ Do you have a link to an example?

Reply:
I added the label to #196, so it should work if you regenerate the changelog now.

Reply:
Awesome, thanks!